nodestream_plugin_neptune/graph_objects.py

```diff
@@ -0,0 +1,112 @@
+import typing as t
+
+class NeptuneNode():
```

Comment:
Confused on the inclusion of these classes. Are they needed? I don't see anywhere where one is created.

Reply:
Removed. Also fixed an import referencing this file, although it was only imported for typing so there shouldn't be any functional changes.